CMakeLists.txt

```diff
@@ -429,27 +431,34 @@ if(WIN32)
         "${OpenMW_SOURCE_DIR}/Docs/license/DejaVu Font License.txt"
         "${OpenMW_BINARY_DIR}/settings-default.cfg"
         "${OpenMW_BINARY_DIR}/gamecontrollerdb.txt"
-        "${OpenMW_BINARY_DIR}/Release/openmw.exe"
         DESTINATION ".")
+	INSTALL(FILES "${OpenMW_BINARY_DIR}/Debug/openmw.exe" DESTINATION "." CONFIGURATIONS Debug)
+	INSTALL(FILES "${OpenMW_BINARY_DIR}/Release/openmw.exe" DESTINATION "." CONFIGURATIONS Release;RelWithDebInfo;MinSizeRel)
```

Comment:
I don't like this duplication of code. At the very least you should set a variable to "Debug" or "Release" depending on current configuration, then use that variable in the install commands. But perhaps there is a better way to check where the executable resides? Have you tried using RUNTIME_OUTPUT_DIRECTORY of the target's properties, or maybe CMAKE_CURRENT_BINARY_DIR ?